src/lib/components/ui/builds/artifact-item.svelte

```diff
@@ -15,11 +15,11 @@
 {#if signature}
 	<div class="flex gap-2 w-full">
 		<img
-			class="flex-grow aspect-square object-contain h-0 min-h-full"
+			class="basis-1/4 aspect-square object-contain w-full"
```

Comment:
Would add size-14 since image is pretty huge.

![Screenshot 2024-04-02 215836](https://github.com/dval-in/dvalin-frontend/assets/14811944/1b43b8ec-dc1f-4e75-ae8a-132e20098ad5)

Reply:
Text is also still clipping on smaller screens

Reply:
What screen size is the text clipping on? I can't seem to get it them to clip when I resize the page.

Reply:
![Screenshot 2024-04-06 152709](https://github.com/dval-in/dvalin-frontend/assets/14811944/8171796f-bc54-4192-a26a-eba1b64ad21f)

Clips on widths larger than 1024px (until screen is large enough)

Reply:
Im using an 21:9 display if this helps

Reply:
Ah I see, it doesn't happen on Firefox. Looks like adding size-14 fixes it but I'll test the rest of the page on Chrome to check that the overflow doesn't happen on anything else.